test/hatrac_config.json

```diff
@@ -6,5 +6,23 @@
     "database_schema": "hatrac",
     "database_max_retries": 5,
     "403_html": "<html><body><h1>Access Forbidden</h1><p>%(message)s</p></body></html>",
-    "401_html": "<html><body><h1>Authentication Required</h1><p>%(message)s</p></body></html>"
+    "401_html": "<html><body><h1>Authentication Required</h1><p>%(message)s</p></body></html>",
+    "s3_config": {
```

Comment:
I think this belongs in a new section of INSTALL.md as an example S3 config rather than in the test file which relates to the TravisCI test recipe.

Reply:
OK.